src/picknik_ur_gazebo_config/config/moveit/picknik_ur_gazebo.srdf.xacro

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--This does not replace URDF, and is not an extension of URDF.
+    This is a format for representing semantic information about the robot structure.
+    A URDF file must exist for this robot as well, where the joints and the links that are referenced are defined
+-->
+<robot name="ur5e" xmlns:xacro="http://wiki.ros.org/xacro">
+    <!--Xacro Parameters-->
+    <xacro:arg name="gripper_name" default="$(arg robotiq_85)"/>
```

Comment:
I don't understand why the default is an additional arg. I expected it to just be 
`<xacro:arg name="gripper_name" default="robotiq_85"/>`

Reply:
oh yeah that is search + replace carelessness. Good catch